src/main/java/net/atos/zac/app/formulieren/FormulierDefinitieRESTService.java

```diff
@@ -27,7 +20,7 @@
 import net.atos.zac.policy.PolicyService;
 
 @Singleton
-@Path("formulierDefinities")
+@Path("formulierdefinities")
```

Comment:
I also fixed this in https://github.com/infonl/dimpact-zaakafhandelcomponent/pull/1714 but fine :-)